pkg/providers/cloudstack/cloudstack.go

```diff
@@ -238,9 +238,9 @@ type ProviderKubectlClient interface {
 
 // NewProvider initializes the CloudStack provider object.
 func NewProvider(datacenterConfig *v1alpha1.CloudStackDatacenterConfig, machineConfigs map[string]*v1alpha1.CloudStackMachineConfig, clusterConfig *v1alpha1.Cluster, providerKubectlClient ProviderKubectlClient, validator ProviderValidator, writer filewriter.FileWriter, now types.NowFunc, log logr.Logger) *cloudstackProvider { //nolint:revive
-	controlPlaneMachineSpec, _ := getControlPlaneMachineSpec(clusterConfig.Spec, machineConfigs)
-	etcdMachineSpec, _ := getEtcdMachineSpec(clusterConfig.Spec, machineConfigs)
-	workerNodeGroupMachineSpecs, _ := getWorkerNodeGroupMachineSpec(clusterConfig.Spec, machineConfigs)
+	controlPlaneMachineSpec := getControlPlaneMachineSpec(clusterConfig.Spec, machineConfigs)
+	etcdMachineSpec := getEtcdMachineSpec(clusterConfig.Spec, machineConfigs)
+	workerNodeGroupMachineSpecs := make(map[string]v1alpha1.CloudStackMachineConfigSpec, len(machineConfigs))
```

Comment:
This was changed in https://github.com/aws/eks-anywhere/pull/5434, but we don't want any side effects happening from this change in the CLI flow, so reverting.